### DISCUSSION TITLE
面試時絕對不要問的一個問題

### DISCUSSION THREAD

**Discussion by ALAN30359704** [1101-GS4719-C/Python-Community-GS4719-C#1246] [General]
找工作面試時，提出哪些問題可能會害你的錄取機率大幅下滑？美國投資公司RSE Ventures執行長希金斯（Matt Higgins）逆風建議求職者，千萬不要在求職面試時詢問雇主，這間公司的遠距工作政策是什麼。

有些人可能會嗆希金斯說，「好了啦，長輩，時代變了」。不過希金斯表示，他非常支持混合辦公與遠距工作，提出這個建議並不是因為他跟一些控制狂老闆一樣，認為員工遠距工作就只會打混摸魚。

希金斯同意遠距工作和混合辦公，可以減少很多在公司無意義的面對面溝通和不必要的人際磨擦，更不用說能讓員工擁有更多生活的主動權，一般來說遠距工作的也會變得比較快樂。

但希金斯認為，求職者必須面對現實，隨著美國聯準會持續升息以遏制通膨，今年美國有數以百萬計的工作陷入危殆。

身為公司執行長、創辦人以及一個投資人，希金斯一路學習著處理不確定性。在接受不確定性後，危機管理的第二條法則是要生存下去，而要在可能很快演變為割喉戰場的求職市場找到工作，需要一些策略和妥協。

老闆內心是怎麼看待遠距工作的？

2022年，GoodHire一項調查發現，78%主管比較偏好面對面進行部分工作。51%主管「肯定會考慮」給拒絕返回公司實體上班的員工減薪。不可避免地，像臉書的祖克柏（Mark Zuckerberg）、特斯拉的馬斯克（Elon Musk），很多領導者和執行長（CEO）都覺得風向已經太偏向支持遠距工作，進而損害了員工的績效。

而且，Meta大規模裁員後，停止提供新進人員遠距工作的選項。

微軟公司最近的工作趨勢調查（Work Trends Study）發現，87%員工在遠距工作或混合工作形態下效率更佳。相反地，85%領導者認為混合工作讓主管更難判斷員工是不是很有生產力。

根據去年公布的數項報告與調查，多數美國員工寧願辭職也不想一週五天回公司上班，但像高盛、摩根士丹利等華爾街大銀行，則命令員工回公司，限制遠距工作政策。根據麥肯錫去年六月的調查，約58%美國人至少每星期可以有一天選擇遠距工作。

主管不會為優秀的人破例

希金斯表示，不管公不公平，很多主管就是認為在求職面試時，最該優先討論的並不是遠距工作政策。希金斯自己遇到面試時劈頭就提問遠距工作方針的求職者時，內心也會立刻警鈴大作。他要求職者們想像一下。求職者問了一個關於遠距工作的問題，一片冷場，面試結束，隔天，人資主管見了另一位願意下週一一早就進辦公室的求職者。

也許你比競爭對手能力更強，但老闆很有可能認定另一位求職者更願意奉獻。即使是在遠距工作政策很完善的公司，主管也會這麼想。

難道你就非得接受實體工作嗎？如果你不願妥協呢？

希金斯表示，你先透過各種網路資源做好功課，並在對方邀你進入第二輪面試時，再提出問題，原因是當他們喜歡上一個求職者時，就會願意做出讓步。你最好給對方留下良好的第一印象，這樣一來你就有一些籌碼可以爭取遠距工作，成為公司的特例。

**Comment by Hahahahbla** [comment]
在面試時，適當避免一些敏感問題，特別是遠距工作政策的問題，可以提高你的錄取機率。先展示你的價值和能力，給面試官留下良好的第一印象，然後在適當的時機再提出你的具體需求，這樣可以更好地平衡求職者的期望和公司的要求。

**Comment by NcuJerryKao** [comment]
這篇文章針對求職面試時討論遠距工作政策的建議，提供了一些有價值的觀點。然而，為了使這篇文章更加全面和具建設性，以下是幾個建議：
1. 增加結構和清晰度：
- 建議將文章分成明確的段落，每個段落集中討論一個主要觀點。例如，先介紹為什麼遠距工作問題在面試中可能不受歡迎，接著提供具體的統計數據來支持這一觀點，然後討論應對策略。
2. 提供更多背景信息：
- 可以增加一些背景信息，說明為什麼遠距工作成為求職者關注的焦點。這可以包括疫情期間遠距工作的普及以及其對工作生活平衡的影響。
3. 增加具體案例：
- 提供一些具體的案例或故事，來說明在面試時討論遠距工作政策的正反面影響。這將使文章更具說服力和實際應用性。
4. 加入專家的建議：
- 除了希金斯的觀點，可以引用其他業界專家或資深人力資源專員的意見，來提供更全面的視角。
5. 強調策略和解決方案：
- 增加更多具體的策略和建議，告訴求職者如果真的想討論遠距工作政策，可以如何在面試過程中巧妙地提出這個問題。例如，可以在面試的最後階段，當面試官詢問是否有其他問題時，再提出這個問題。
6. 平衡觀點：
- 雖然文章強調不要在面試時立即詢問遠距工作政策，但也可以平衡一下，討論一些情況下這樣做可能是合適的。例如，如果公司在招聘廣告中強調遠距工作，求職者在面試時詢問這個政策可能是合理且必要的。
7. 語氣調整：
- 語氣可以稍微調和一下，避免過於絕對。例如，可以說「在大多數情況下，建議不要一開始就詢問遠距工作政策」，而不是「千萬不要在求職面試時詢問雇主，這間公司的遠距工作政策是什麼。」
8. 結論總結：
- 最後可以加上一個總結段落，重申主要觀點並提供一些正面的行動建議，讓讀者有一個清晰的記憶點。
透過這些建議，文章將更具結構性、信息豐富且實用，使讀者更容易理解和應用其中的建議。

**Comment by titan92601** [comment]
我認為提出公司的遠距工作政策可能在面試中損害錄取機會。即使支持遠距工作，主管也更重視其他方面，如求職者的奉獻度。建議先了解公司情況，待第二輪面試時再提出問題，以增加特例遠距工作的機會。

**Comment by sandyho0307** [comment]
希金斯的建議反映了許多雇主和企業領導者在遠距工作問題上的真實想法。雖然遠距工作在現代勞動市場中越來越普及，許多員工也證實了其對工作效率和生活質量的正面影響，但許多企業領導仍對此持保留態度。

**Comment by OHoln** [comment]
的確，比起遠距工作的政策在面試中可能有更多對面試官來說優先度更高、更能衡量你的能力及特質的問題，想提出這樣的問題可得仔細斟酌時機

**Comment by 710388** [comment]
這是什麼神奇的問題，也太直接了，不是很禮貌的感覺
